root_orchestrator/cloud-scheduler/Dockerfile

```diff
@@ -19,6 +19,9 @@ ENV SYSTEM_MANAGER_PORT=10000
 ENV CLOUD_MONGO_URL=131.159.24.210
 ENV CLOUD_MONGO_PORT=10007
 
+ENV RESOURCE_ABSTRACTOR_URL=131.159.24.210
```

Comment:
Just a guess here. Would the upcoming `abstractor-service` container run there? @giobart